src/app/todos/new-todo-form/new-todo-form.component.html

```diff
@@ -0,0 +1,6 @@
+<form class="mt-4 justify-content-between form" [formGroup]="todoForm" (ngSubmit)="onSubmit()">
+  <input class="" type="text" id="title" formControlName="title" name="title" required>
+  <button class="btn btn-primary" type="submit">Add Todo</button>
+</form>
```

Comment:
Totally up to you, but it is possible to simply create a FormControl in the .ts file, without creating an entire FormGroup, if you have just one input. 

In this case you won't be specifying `formControlName` on the input, but `[FormControl]="createdFormControlFromTsFile"`

And then adding an onClick handler to the button. It's just so you know  you can do that as well :)